cmd/edv-rest/startcmd/start_test.go

```diff
@@ -259,34 +283,53 @@ func TestListenAndServe(t *testing.T) {
 }
 
 func TestActualDbInitializer_CreateConfigStore(t *testing.T) {
```

Comment:
Now that the DB initializer is gone, this should be renamed

Reply:
fixed